core/src/main/resources/error/error-classes.json

```diff
@@ -792,6 +887,21 @@
       "Unable to acquire <requestedBytes> bytes of memory, got <receivedBytes>"
     ]
   },
+  "UNABLE_TO_CONVERT_TO_PROTOBUF_TYPE" : {
+    "message" : [
+      "Unable to convert SQL type <toType> to Protobuf type <protobufType>."
+    ]
+  },
+  "UNABLE_TO_LOCATE_PROTOBUF_MESSAGE_ERROR" : {
```

Comment:
```suggestion
  "PROTOBUF_MESSAGE_NOT_FOUND" : {
```

Reply:
@srielau fixed all the error class names